Hestia/Hestia/frontend/DevicesScreen/UITableViewControllerDevicesMain.cs

```diff
@@ -316,6 +324,61 @@ public void ProcessSpeech(string result)
                     WarningMessage.Display(strings.noDeviceFound, strings.pronounceDeviceNameCorrectly, this);
                 }
             }

```

Comment:
I think this does not address the comment of Feiko : https://github.com/RUGSoftEng/2018-Hestia-IOS/pull/221#discussion_r192992669

Reply:
OK, can you then reply that to his comment, such that he knows why it will not be changed?